java/core/pom.xml

```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>com.lancedb</groupId>
         <artifactId>lancedb-parent</artifactId>
-        <version>0.0.3</version>
+        <version>0.10.0-beta.1</version>
```

Comment:
naming convention here is alpha/beta/release-candidate? if not i think -SNAPSHOT should be suffice?

Reply:
yep, this is the lancedb naming convention